packages/xo-web/src/common/render-xo-item.js

```diff
@@ -143,6 +143,38 @@ PoolItem.propTypes = XO_ITEM_PROP_TYPES
 
 // ===================================================================
 
+export const SrResourceSetItem = [
+  connectStore(() => {
+    const getSr = createGetObject()
+    const getContainer = createGetObject((state, props) => {
+      const sr = getSr(state, props, true) // true to bypass view permissions
+      return sr && sr.$container
+    })
+
+    return (state, props) => ({
+      sr: getSr(state, props, true),
+      container: getContainer(state, props, true),
```

Comment:
After thinking about it, I don't think we should show the container at all to the self users.